mautic/mautic
# Api fixes

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | Y
| New feature? | N
| Automated tests included? | Y
| Related user documentation PR URL | /
| Related developer documentation PR URL | /
| Issues addressed (#s or URLs) | /
| BC breaks? | N
| Deprecations? | N

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

Running the API Library's functional test discovered many bugs. Fixing them here.

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Run tests from https://github.com/mautic/api-library against a Mautic instance on the 3.x branch
2. See the errors

#### Steps to test this PR:
1. Run the tests again on this branch but also check out the [API Library 3.0 branch](https://github.com/mautic/api-library/tree/3.0) before you run the tests as it has deprecated methods removed.
2. Should see no errors
